packages/cli/src/commands/runAndroid/__tests__/sampleGeneratedDebugManifest.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this could also be moved to `__fixtures__`

Reply:
Done